packages/fxa-content-server/tests/functional/settings_v2/lib/helpers.js

```diff
@@ -33,6 +38,62 @@ async function navigateToSettingsV2(remote) {
   return email;
 }
 
+async function generateSecurityCode(remote) {
+  await click(
+    selectors.SETTINGS_V2.SECURITY.TFA.ADD_BUTTON,
+    selectors.SETTINGS_V2.SECURITY.TFA.SECURITY_CODE_TEXTBOX_LABEL,
+    remote
+  );
+  await click(selectors.SETTINGS_V2.SECURITY.TFA.SHOW_CODE_LINK, remote);
+
+  // store the secret key to use later
+  await testElementExists(
+    selectors.SETTINGS_V2.SECURITY.TFA.SECRET_CODE_MODAL,
+    remote
+  );
+  secret = await remote
+    .findByCssSelector(selectors.SETTINGS_V2.SECURITY.TFA.SECRET_CODE_TEXT)
```

Comment:
nit: I'm surprised we didn't have a helper method for "getElementText", welcome to add if you think its useful.